tests/integration_tests/performance/test_vsock_ab.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the VSOCK throughput of Firecracker uVMs."""
+
+import os
+
+import pytest
+
+from framework.utils import CpuMap
+from framework.utils_iperf import IPerf3Test, emit_iperf3_metrics
+from framework.utils_vsock import VSOCK_UDS_PATH, make_host_port_path
+
+BASE_PORT = 5201
+
+# How many clients/servers should be spawned per vcpu
+LOAD_FACTOR = 1
+
+# Time (in seconds) for which iperf "warms up"
+WARMUP_SEC = 3
+
+# Time (in seconds) for which iperf runs after warmup is done
+RUNTIME_SEC = 20
+
+# VM guest memory size
+GUEST_MEM_MIB = 1024
```

Comment:
Can we move these constants inside `VsockIperf3Test` like what you do for `TCPIPerf3Test`?

Reply:
Sure! :)